bin/validateSchema.js

```diff
@@ -5,47 +5,74 @@ import yaml from 'js-yaml';
 import { convertOpenApiToJsonSchema } from '../utils/convertOpenApiToJsonSchema.js';
 import 'dotenv/config';
 
-const apiKey = process.env.PRIVATE_KEY;
-const visitorId = process.env.VISITOR_ID;
+function gatherEnvs() {
+  const envs = {
+    PRIVATE_KEY: process.env.PRIVATE_KEY,
+    VISITOR_ID: process.env.VISITOR_ID,
+    //REQUEST_ID: process.env.REQUEST_ID,
```

Comment:
Looks like something missed

Reply:
Yes, It's disabled until I get `requestId` from Gabe.